config/locales/views/en.yml

```diff
@@ -380,3 +380,18 @@ en:
       title: "Invitation Status for %{email}"
       position_html: "You are currently number %{position} on our waiting list!"
       date: "At our current rate, you should receive an invitation on or around: %{date}."
+  pseuds:
+    delete_preview:
+      heading: "Pseud deletion"
+      notice: "When you delete your pseud, any works, series, or comments you have created under it will be transferred to your default pseud."
+      saved_bookmarks: 
+        one: "You have saved %{count} bookmark using this pseud. You can choose whether to delete them or transfer them to your default pseud."
```

Comment:
Not to step on @redsummernight's toes here, but the second sentence here should probably use "it" rather than "them," since we're just talking about one bookmark. It also looks like this file needs a newline at the end.

Reply:
My bad!